src/core/api/base.go

```diff
@@ -197,6 +199,8 @@ func Init() error {
 
 	retentionController = retention.NewAPIController(retentionMgr, projectMgr, repository.Mgr, scheduler.Sched, retentionLauncher)
 
+	gcController = gc.NewController()
```

Comment:
You can use the global gc controller defined in the package `controller/gc` so that we don't need to declare a new one?

Reply:
updated.